wordle-web/src/pages/game.vue

```diff
@@ -0,0 +1,26 @@
+<template>
```

Comment:
This file should be under your views, which is basically just your pages folder.